nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetHTTP.java

```diff
@@ -467,6 +486,18 @@ public void onTrigger(final ProcessContext context, final ProcessSessionFactory
             if (accept != null) {
                 get.addHeader(HEADER_ACCEPT, accept);
             }
+
+            // Add dynamic headers
+
+            PropertyValue customHeaderValue;
+            for (PropertyDescriptor customProperty : customHeaders) {
+                customHeaderValue = context.getProperty(customProperty).evaluateAttributeExpressions();
```

Comment:
I could see this potentially making use of the incoming FlowFile as seed data for the EL.  Perhaps we should pass in the flowFile?